daq_macros.py

```diff
@@ -3112,7 +3112,7 @@ def zebraDaqBluesky(flyer, angle_start, scanWidth, imgWidth, exposurePeriodPerIm
     x_beam = getPvDesc("beamCenterX")
     y_beam = getPvDesc("beamCenterY")
     if beamline == "nyx":
-      wavelength = daq_utils.energy2wave(beamline_lib.motorPosFromDescriptor("energy"))
+      wavelength = daq_utils.energy2wave(beamline_lib.motorPosFromDescriptor("energy"), digits=6)
```

Comment:
Above it's just "6" rather than  "digits=6", is either okay? Unless there's a need, I'd suggest keeping them identical regardless.